waku/v2/protocol/filter/filter_push_test.go

```diff
@@ -141,3 +144,84 @@ func (s *FilterTestSuite) TestLargePayloadsUTF8() {
 	s.Require().NoError(err)
 
 }
+
+func (s *FilterTestSuite) TestTimestampInFuture() {
+
+	// Subscribe
+	s.subDetails = s.subscribe(s.testTopic, s.testContentTopic, s.fullNodeHost.ID())
+
+	// Set time in the future
+	timeDelta, _ := time.ParseDuration("200h")
+	futureTime := proto.Int64(time.Now().UnixNano() + timeDelta.Nanoseconds())
+	message := tests.CreateWakuMessage(s.testContentTopic, futureTime, "test_payload")
+
+	// Should get accepted
+	_, err := s.relayNode.Publish(s.ctx, message, relay.WithPubSubTopic(s.testTopic))
```

Comment:
Not able to understand the objective of this test. 

Maybe this test should be put under RLNRelay where publish should get failed.

Reply:
You are right, not much value of these tests without RLNRelay. I've removed them from PR. I'll study RLNRelay and reuse them later.